pennylane/pulse/transmon.py

```diff
@@ -241,9 +241,9 @@ def transmon_drive(amplitude, phase, freq, wires, d=2):
     in :class:`ParametrizedHamiltonian` and have a signature of two parameters, ``(params, t)``.
 
     For realistic simulations, one may restrict the amplitude, phase and drive frequency parameters.
-    For example, the authors in `2008.04302 <https://arxiv.org/abs/2008.04302>`_ impose the restrictions of
-    a maximum amplitude :math:`\Omega_{\text{max}} = 20 \text{MHz}` and the carrier frequency to deviate at most
-    :math:`\nu - \omega = \pm 1 \text{GHz}` from the qubit frequency :math:`\omega`
+    For example, the authors in `2203.06818 <https://arxiv.org/abs/2203.06818>`_ impose the restrictions of
+    a maximum amplitude :math:`\Omega_{\text{max}} = 0.02 \times 2 \pi \text{GHz}` and the carrier frequency to deviate at most
+    :math:`\nu - \omega = \pm 1 \times 2 \pi \text{GHz}` from the qubit frequency :math:`\omega`
```

Comment:
What do we think the authors in [2203.06818](https://arxiv.org/abs/2203.06818) meant here? $20 \times 10^6 \text{rad/s}$ or $20 \times 2 \pi \text{MHz}$?

![image](https://user-images.githubusercontent.com/43949391/235138432-fcf4feda-81a8-4aed-97f3-a184c1469a9d.png)

Reply:
Isnt it even okay to write Hz here?